frontend/src/components/adjudications/AdjudicateForm.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { TextField } from '@material-ui/core';
+import Button from '../interface/Button';
+import CheckBox from '../interface/CheckBox';
+import PropTypes from 'prop-types';
+import { useState, useEffect } from 'react';
+import { awardConsiderationEnum } from '../../constants';
+import { updateAdjudications } from "../../api/AdjudicationAPI";
+import { getAdjudications } from '../../api/AdjudicationAPI';
+import { getPerformance } from '../../api/PerformanceAPI';
+import humps from 'humps';
+
+export default function AdjudicateForm(props) {
+    const { match: { params: { eventId, performanceId }}} = props;
+    const [loading, setLoading] = useState(true)
+    const [adjudications, setAdjudications] = useState({}) 
+    const [performanceValues, setPerformancesValues] = useState({}) 
+
+//according to docs, componentDidMount() is similar to useEffect(() => {}); 
```

Comment:
nitpick: align the comments and closing bracket for this function, and indent the `.then`s and corresponding brackets